### PR TITLE
Update ASOv2 samples

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

**Comment by matthchr** [comment]
@babbageclunk - were there any modifications you needed to make to these? I can either integrate them here or you can send a follow up PR

**Review by theunrepentantgeek — APPROVED** [review]
Looks good to me. :shipit: 

gif

File: hack/generated/config/samples/microsoft.compute/v1alpha120201201_virtualmachinescaleset.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+spec:
+  location: westcentralus
+  owner:
+    name: aso-sample-rg
+  sku:
+    name: standard_d1_v2
+    capacity: 1
+  platformFaultDomainCount: 3
+  singlePlacementGroup: false
+  upgradePolicy: Automatic
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.compute/v1alpha120201201_virtualmachinescaleset.yaml:15]:
```suggestion
  upgradePolicy: 
    mode: Automatic
```

File: hack/generated/config/samples/microsoft.compute/v1alpha120201201_virtualmachinescaleset.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              path: /home/adminUser/.ssh/authorized_keys
+    networkProfile:
+      networkInterfaceConfigurations:
+        - name: samplenicconfig
+          primary: true
+          ipConfigurations:
+            - name: sampleipconfiguration
+              subnet:
+                reference:
+                  group: microsoft.networking
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.compute/v1alpha120201201_virtualmachinescaleset.yaml:41]:
```suggestion
                  group: microsoft.network.infra.azure.com
```

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+apiVersion: microsoft.networking.infra.azure.com/v1alpha1api20201101
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml:1]:
```suggestion
apiVersion: microsoft.network.infra.azure.com/v1alpha1api20201101
```

And the directory should be `microsoft.network` for consistency.

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  location: westcentralus
+  owner:
+    name: aso-sample-rg
+  sku:
+    name: Standard
+  frontendIPConfigurations:
+    - name: LoadBalancerFrontend
+      publicIPAddress:
+        reference:
+          group: microsoft.networking
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml:16]:
```suggestion
          group: microsoft.network.infra.azure.com
```

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_publicipaddress.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+apiVersion: microsoft.networking.infra.azure.com/v1alpha1api20201101
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.networking/v1alpha120201101_publicipaddress.yaml:1]:
```suggestion
apiVersion: microsoft.network.infra.azure.com/v1alpha1api20201101
```

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_virtualnetwork.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+apiVersion: microsoft.networking.infra.azure.com/v1alpha1api20201101
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.networking/v1alpha120201101_virtualnetwork.yaml:1]:
```suggestion
apiVersion: microsoft.network.infra.azure.com/v1alpha1api20201101
```

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_virtualnetworksubnet.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+apiVersion: microsoft.networking.infra.azure.com/v1alpha1api20201101
```
Comment (babbageclunk) [inline hack/generated/config/samples/microsoft.networking/v1alpha120201101_virtualnetworksubnet.yaml:1]:
```suggestion
apiVersion: microsoft.network.infra.azure.com/v1alpha1api20201101
```

**Review by babbageclunk — APPROVED** [review]
These were super useful in recording the demo, thanks! I've commented on the things I needed to tweak to get them deployed.

File: hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml (hunk 3 above)
Comment (matthchr) [reply hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml:1]: Thanks!

**Pull request merged**